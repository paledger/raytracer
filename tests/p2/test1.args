pixelcolor ../resources/simple.pov 640 480 275 285
