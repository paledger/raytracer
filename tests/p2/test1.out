Pixel: [275, 285] Ray: {0 0 14} -> {-0.0919 0.09397 -0.9913}
T = 13.1
Object Type: Sphere
BRDF: Blinn-Phong
Color: (203, 0, 203)
